docs/docs/sample-datasets.md

```diff
@@ -48,25 +48,27 @@ ds = xr.open_dataset(
 )
 ```
 
-<!-- === "Cloudflare R2" -->
+=== "Cloudflare R2"
 
-<!-- ```python -->
-<!-- import icechunk as ic -->
-<!-- import xarray as xr -->
+```python
+import icechunk as ic
+import xarray as xr
 
-<!-- storage = ic.s3_storage( -->
-<!--     bucket="icechunk-public-data", -->
-<!--     prefix="v1/era5_weatherbench2", -->
-<!--     endpoint_url="foo", -->
-<!--     region="auto", -->
-<!-- ) -->
+storage = ic.s3_storage(
+    bucket="v1",
+    prefix="era5_weatherbench2",
+    endpoint_url="https://data.icechunk.cloud",
+    region="auto",
+    force_path_style=True,
+)
```

Comment:
I think we should provide this user API
```python
ic.r2_storage(
    prefix="v1/era5_weatherbench2",
    endpoint_url="https://data.icechunk.cloud/",
)
```

and set the `bucket`, `region` and `force_path_style` ourselves.

Reply:
I agree, particularly to isolate us from futures change to the AWS SDK